_posts/2016-12-04-encore.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: ENCORE ensemble similarity
+---
+
+The **ENCORE** ensemble similarity library has been integrated in the next
+version of MDAnalysis
+as
+[MDAnalysis.analysis.encore](http://www.mdanalysis.org/mdanalysis/documentation_pages/analysis/encore.html).
+It implements a variety of techniques for calculating similarities between
+structural ensembles (trajectories), as described in this publication:
+
+Tiberti M, Papaleo E, Bengtsen T, Boomsma W, Lindorff-Larsen K (2015), ENCORE:
+Software for Quantitative Ensemble Comparison. PLoS Comput Biol 11(10):
+e1004415.
+doi:[10.1371/journal.pcbi.1004415](http://doi.org/10.1371/journal.pcbi.1004415).
+
+Using the similarity measures is simply a matter of loading the trajectories or
+experimental ensembles that one would like to compare as MDAnalysis.Universe
+objects:
+
+```python
+>>> from MDAnalysis import Universe
+>>> import MDAnalysis.analysis.encore as encore
+>>> from MDAnalysis.tests.datafiles import PSF, DCD, DCD2
+>>> u1 = Universe(PSF, DCD)
+>>> u2 = Universe(PSF, DCD2)
+```
+
+and running the similarity measures on them, choosing among the Harmonic
+Ensemble Similarity measure:
+
+```python
+>>> hes_similarities, details = encore.hes([u1, u2])
+>>> print hes_similarities
+[[        0.         38279683.9587939]
+ [ 38279683.9587939         0.       ]]
+```
+
+Similarities are written in a square symmetric matrix having the same dimensions
+and ordering as the input list, with each element being the similarity value for
+a pair of the input ensembles. Other available measures are the clustering
+ensemble similarity measure `encore.ces` and dimensionality reduction ensemble
+measure `encore.dres`.
+
+The encore library includes a general interface to various clustering and
+dimensionality reduction algorithms (through
+the [scikit-learn](http://scikit-learn.org/) package), which makes it easy to
+switch between clustering and dimensionality reduction algorithms when using the
+`ces` and `dres` functions. The clustering and dimensionality reduction
+functionality is also directly available through the `cluster` and
+`reduce_dimensionality` functions. For instance, to cluster the conformations
+from the two universes defined above, we can write:
+
+```python
+>>> cluster_collection = encore.cluster([u1,u2])
+>>> print cluster_collection
+0 (size:5,centroid:1): array([ 0,  1,  2,  3, 98])
+1 (size:5,centroid:6): array([4, 5, 6, 7, 8])
+2 (size:7,centroid:12): array([ 9, 10, 11, 12, 13, 14, 15])
```

Comment:
The blog doesn't really explan the output. It suggests I could infer from it to which trajectory a centroid belongs to.